src/components/Table/index.tsx

```diff
@@ -138,9 +138,9 @@ class Table<T> extends React.Component<TableProps<T>, TableState<T>> {
 		};
 	}
 
-	componentWillReceiveProps(newProps: TableProps<T>) {
-		if (newProps.sort !== this.props.sort) {
-			this.setState({ sort: newProps.sort } as TableState<T>);
+	componentDidUpdate(prevProps: TableProps<T>) {
```

Comment:
This one can easily be done using `getDerivedStateFromProps`, something like:
```
static getDerivedStateFromProps(nextProps, prevState){
        if (nextProps.sort !== prevState.list) {
            return { sort: nextProps.sort };
        }
        
      return null;
    }
```

This also means we can add a check if state is undefined and set the default value that is currently set in the constructor.

Reply:
`getDerivedStateFromProps` havent `prevProps`, and that's why we can't use it